cpp/models/abm/testing_strategy.cpp

```diff
@@ -177,6 +177,8 @@ bool TestingStrategy::run_strategy(PersonalRandomNumberGenerator& rng, Person& p
         return true;
     }
 
+    bool is_person_compliant_to_test = person.is_compliant(rng, InterventionType::Testing);
+
```

Comment:
I think this can be used as an early return, like `if (not compliant) return false`, since by line 193
> // If the Person does not comply to Testing where there is a testing scheme at the target location, it is not allowed to enter.

Reply:
Yes, I agree. The code is updated now!